cogs5e/initTracker.py

```diff
@@ -925,7 +925,8 @@ async def effect(self, ctx, target_name: str, effect_name: str, *args):
         -neutral <damage type> - Removes the combatant's immunity, resistance, or vulnerability to the given damage type.
         __General__
         -ac <ac> - modifies ac temporarily; adds if starts with +/- or sets otherwise.
-        -sb <save bonus> - Adds a bonus to all saving throws."""
+        -sb <save bonus> - Adds a bonus to all saving throws.
+        -desc <"description"> - Adds a description of the effect."""
```

Comment:
why the quotes here?

Reply:
To represent that you need quotes if its more than one word. Followed the format from the attack arg in `!init effect`

Reply:
I guess. Feels a little weird, though, since usually the requirement for quotes in an argument is implicit (see !c -phrase for example - they're only there in the attack arg because it has a format that needs to be followed - honestly I'm not 100% on that docs format either, but eh)